### PR TITLE
Pods 5284 Scheme migration - Add Trustees name | Individual

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/helpers/BeforeYouStartCYAHelper.scala [hunk 1]
```diff
@@ -73,9 +88,8 @@ class BeforeYouStartCYAHelper extends CYAHelper with CountriesHelper {
         answerTransform = Some(booleanToContent)
       )
     )
+    val beforeYouStart = s1 ++ s2 ++ s3
```
Comment (vinnicombe) [inline app/helpers/BeforeYouStartCYAHelper.scala:91]: Can these be named more meaningfully?

File: app/utils/datacompletion/DataCompletion.scala [hunk 2]
```diff
@@ -34,8 +35,12 @@ trait DataCompletion {
       case (_, false) => false
     })
 
-  def isBeforeYouStartCompleted: Boolean =
-    !List(get(SchemeNameId), get(SchemeTypeId), get(EstablishedCountryId)).contains(None) && get(WorkingKnowledgeId).nonEmpty
+  def isBeforeYouStartCompleted: Boolean = {
+    (get(SchemeTypeId), get(HaveAnyTrusteesId)) match {
+      case (Some(st), None) if st == SchemeType.BodyCorporate || st == SchemeType.GroupLifeDeath => false
+      case _ => !List(get(SchemeNameId), get(SchemeTypeId), get(EstablishedCountryId)).contains(None) && get(WorkingKnowledgeId).nonEmpty
```
Comment (vinnicombe) [inline app/utils/datacompletion/DataCompletion.scala:41]: Negating this check is clunky, can forall/exists be used to make this easier to read?

File: app/helpers/BeforeYouStartCYAHelper.scala [hunk 3]
```diff
@@ -73,9 +88,8 @@ class BeforeYouStartCYAHelper extends CYAHelper with CountriesHelper {
         answerTransform = Some(booleanToContent)
       )
     )
+    val beforeYouStart = section1 ++ section2 ++ section3
```
Comment (vinnicombe) [inline app/helpers/BeforeYouStartCYAHelper.scala:91]: This is no more descriptive than before! I have to read each answer row contents and try and work out what they are. Can they be more descriptive but still with some brevity?

File: app/identifiers/trustees/TrusteesId.scala [hunk 4]
```diff
@@ -20,9 +20,11 @@ import identifiers.TypedIdentifier
 import play.api.libs.json._
 
 case class TrusteesId(index: Int) extends TypedIdentifier[Nothing] {
-  override def path: JsPath = __ \ TrusteesId.toString \ index
+  override def path: JsPath = TrusteesId.collectionPath \ index
 }
 
 object TrusteesId {
+  val collectionPath: JsPath = __ \ TrusteesId.toString
```
Comment (vinnicombe) [inline app/identifiers/trustees/TrusteesId.scala:27]: As per previous PRs, is this needed?

Comment (grantrobson) [reply app/identifiers/trustees/TrusteesId.scala:27]: yes this is used for cleanup

Comment (vinnicombe) [reply app/identifiers/trustees/TrusteesId.scala:27]: Ok cool

**Pull request merged**